src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/DebugPropertyPage/RemoveStartArgumentsActionHandler.cs

```diff
@@ -0,0 +1,96 @@
+using Microsoft.VisualStudio.ProjectSystem.VS.PropertyPages.Designer;
+using Microsoft.VisualStudio.Threading;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Properties;
+
+[Export(typeof(ILinkActionHandler))]
+[ExportMetadata("CommandName", "RemoveStartArguments")]
+internal class RemoveStartArgumentsActionHandler : ILinkActionHandler
+{
+    private const string PropertyCommandArgsInUserFile = "StartArguments";
+    private const string UserSuffix = ".user";
+
+    private string? _xmlFile;
+    private List<string>? userFileCommandArgs;
+
+    public async Task HandleAsync(UnconfiguredProject project, IReadOnlyDictionary<string, string> editorMetadata)
+    {
+        _xmlFile = project.FullPath + UserSuffix;
+
+        await RemoveElementAsync();
+
+        await CopyUserArgsToLaunchProfileAsync(project);
+    }
+
+    private async Task RemoveElementAsync()
+    {
+        await TaskScheduler.Default;
+
+        XmlDocument doc = new() { XmlResolver = null };
+
+        try
+        {
+#pragma warning disable CA3075 // Insecure DTD processing in XML
+            doc.Load(_xmlFile);
+#pragma warning restore CA3075 // Insecure DTD processing in XML
+        }
+        catch (Exception ex)
+        {
+            TraceUtilities.TraceException("Failed to load .user file", ex);
+        }
+
+        XmlNodeList xmlNodeList = doc.GetElementsByTagName(PropertyCommandArgsInUserFile);
+
+        for (int i = xmlNodeList.Count - 1; i >= 0; --i)
+        {
+            userFileCommandArgs ??= new();
+            userFileCommandArgs.Add(xmlNodeList[i].InnerText);
+            xmlNodeList[i].ParentNode.RemoveChild(xmlNodeList[i]);
+        }
+
+        try
+        {
+            doc.Save(_xmlFile);
+        }
+        catch (Exception ex)
+        {
+            TraceUtilities.TraceException("Failed to save .user file", ex);
+        }
+
+        return;
+    }
+
+    private async Task CopyUserArgsToLaunchProfileAsync(UnconfiguredProject project)
+    {
+        ILaunchSettingsProvider? launchSettingsProvider = project.Services.ExportProvider.GetExportedValue<ILaunchSettingsProvider>();
+
+        ILaunchSettings launchSettings = await launchSettingsProvider.WaitForFirstSnapshot();
+
+        IWritableLaunchSettings writableLaunchSettings = launchSettings.ToWritableLaunchSettings();
+
+        if (SetPropertyValue(writableLaunchSettings))
+        {
+            await launchSettingsProvider.UpdateAndSaveSettingsAsync(writableLaunchSettings.ToLaunchSettings());
+        }
+    }
+
+    private bool SetPropertyValue(IWritableLaunchSettings writableLaunchSettings)
+    {
+        if (writableLaunchSettings is null || writableLaunchSettings.ActiveProfile is null || userFileCommandArgs is null)
+        {
+            return false;
+        }
+
+        foreach (string? commandArg in userFileCommandArgs)
+        {
+            writableLaunchSettings.ActiveProfile.CommandLineArgs += " " + commandArg;
```

Comment:
Also, this is going to move the `StartArguments` to the command line arguments of the _active_ profile. I see two _potential_ problems:
1. The user may not be editing the active profile. In that case the arguments will just seem to disappear when they click the link.
2. `StartArguments` potentially affect launch/debug regardless of the launch profile, but this will move them to one specific launch profile.

I'm not sure either of those is really a problem, but we should think about those scenarios.